package.json

```diff
@@ -36,64 +36,54 @@
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
     "lodash.omit": "^4.5.0",
-    "pdfjs-dist": "3.4.120",
+    "pdfjs-dist": "3.9.179",
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "provinces": "^1.11.0",
     "qrcode.vue": "1.7.0",
     "regenerator-runtime": "^0.13.11",
     "register-service-worker": "^1.7.2",
     "sbc-common-components": "3.0.8",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-router": "^3.6.5",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
     "vuetify": "^2.6.15",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@pinia/testing": "^0.0.16",
-    "@types/jest": "^27.5.2",
     "@types/lodash": "^4.14.194",
-    "@types/pdfjs-dist": "^2.10.378",
     "@types/sinon": "^10.0.14",
     "@types/vue-the-mask": "^0.11.1",
     "@types/vuelidate": "^0.7.16",
     "@typescript-eslint/eslint-plugin": "^5.59.2",
     "@typescript-eslint/parser": "^5.59.2",
+    "@vitejs/plugin-vue2": "^2.2.0",
+    "@vitest/coverage-v8": "^0.34.2",
```

Comment:
👍 

Smells like we need another ticket to update Create and Edit UIs.

Reply:
Awesome! 🙌

Reply:
Are you creating the new ticket or am I? Or do you want to make the change as part of 17164?

Reply:
New tickets:
https://github.com/bcgov/entity/issues/17495 and https://github.com/bcgov/entity/issues/17497

I added the good first issue label to them also. Let me know if you disagree Sev.